main.js

```diff
@@ -0,0 +1,379 @@
+    $(".resetGame").click(resetGame);
+    $(".playerCharacterSelectionModal img").click(characterClicked);
+    $(".gameStart").click(gameStart);
+}
+
+function gameStart() {
+    $(".gameStart").addClass('hiddenElement'); 
+    $(".playerCharacterSelectionModal").removeClass('hiddenElement');
+    bgMusicPlay();
+}
+
+
+function columnClicked(){
+    if(!screenClickable){//if screenClickable = true then you can click.
+        return;
+    }
+
+    var columnClicked = $(this).attr("column");
+
+    columnClicked = parseInt(columnClicked);
+    //drop token and update game board. //if pop up modal for sorry try again.
+    var currentRowDroppedIn = dropTokenCol( currentPlayer, columnClicked);
+    if (currentRowDroppedIn === null){
+        return;
+    }
+    //check to see if powerup is allowed OR if player has won.
+    tokenPlacementCheck( currentPlayer, columnClicked, currentRowDroppedIn );
+}
+
+function powerupButtonClicked(){
+    //activates powerUp if currentplayer powerupheld is = true;
+    var test = $(this).hasClass(currentPlayer.characterType.characterToken);
+    if(currentPlayer.powerupHeld && test) {
+        usePowerup(currentPlayer.characterType.characterPowerup);
+    
+        currentPlayer.powerupHeld = false;
+        if(currentPlayerStatus) {
+            $('#player1 img').removeClass('animatePowerupButton');
+        }else{
+            $('#player2 img').removeClass('animatePowerupButton');
+        }
+        updateDOM();
+    }
+}
+
+function characterClicked() {
+    if($(this).hasClass("character")) {
+        var characterClicked = $(this).attr("name");
+        if(currentPlayerStatus){
+            player1 = new Player(characterClicked, characters[characterClicked]);
+            player1.characterType.characterSound1.play();
+            $(this).addClass("hiddenElement");
+            $("#player1 img").addClass(characterClicked);
+            $(".playerCharacterSelectionModal h1").text('Player 2, Choose your character!');
+        } else {   
+            player2 = new Player(characterClicked, characters[characterClicked]);
+            player2.characterType.characterSound1.play();
+            $(this).addClass("hiddenElement");
+            $("#player2 img").addClass(characterClicked);
+            $(".playerCharacterSelectionModal h1").text('Let\'s Play!');
+            setTimeout(function(){
+                $(".playerCharacterSelectionModal").addClass('hiddenElement');
+                togglePlayerTurn();
+                }, 2000);
+            
+        }
+        currentPlayerStatus = !currentPlayerStatus;    
+    } 
+}
+// Player Turn Toggle
+function togglePlayerTurn(){
+    //current player has finish his/her turn and currentplayer switches before toggle is called.
+    //if player one finish his turn. then toggle switches to player 2 and calls the modal and sets current player.
+    cancelHurryUp();
+    setTimeout(function(){
+      if(currentPlayerStatus){
```

Comment:
since this is the number, you can make use of that in a lot of your selectors.

Though really, each object should know which dom elements belong to itself.